### PR TITLE
[HOPSWORKS-2071] Data validation

### PR DESCRIPTION
**Pull request by tkakantousis**

### CODE REVIEW

File: files/default/sql/ddl/2.2.0__initial_tables.sql [hunk 1]
```diff
@@ -1965,6 +1966,20 @@ CREATE TABLE `remote_group_project_mapping` (
   CONSTRAINT `fk_remote_group_project_mapping_1` FOREIGN KEY (`project`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
 
+CREATE TABLE  IF NOT EXISTS `feature_group_validation` (
```
Comment (robzor92) [inline files/default/sql/ddl/2.2.0__initial_tables.sql:1969]: The changes are really hard to follow due to the indentation, can you fix it first?

**Review by robzor92 — CHANGES_REQUESTED** [review]

File: files/default/sql/ddl/2.2.0__initial_tables.sql (hunk 1 above)
Comment (tkakantousis) [reply files/default/sql/ddl/2.2.0__initial_tables.sql:1969]: fixed

File: files/default/sql/ddl/updates/2.2.0.sql [hunk 2]
```diff
@@ -9,3 +9,58 @@ ALTER TABLE `users` DROP COLUMN `security_question`, DROP COLUMN `security_answe
 
 ALTER TABLE `hopsworks`.`feature_store_tag` DROP COLUMN `type`;
 ALTER TABLE `hopsworks`.`feature_store_tag` ADD COLUMN `tag_schema` VARCHAR(13000) NOT NULL DEFAULT '{"type":"string"}';
+CREATE TABLE IF NOT EXISTS `validation_rule` (
```
Comment (robzor92) [inline files/default/sql/ddl/updates/2.2.0.sql:12]: Identation is bit off here too

**Review by robzor92 — CHANGES_REQUESTED** [review]

File: files/default/sql/ddl/updates/2.2.0.sql (hunk 2 above)
Comment (tkakantousis) [reply files/default/sql/ddl/updates/2.2.0.sql:12]: fixed

**Review by robzor92 — APPROVED** [review]
